jxon.js

```diff
@@ -76,6 +81,56 @@
       }
     };
 
+    /**
+     * Helper function: given a xml prolog (eg: <?xml version="1.0"?>),
+     * returns a object (eg: { version: "1.0" }).
+     */
+    function getPrologFromString(sXmlProlog) {
+      var matches = sXmlProlog.match(rXmlPrologAttributes);
+      var oReturn = {};
+      for (var i = 0; i < matches.length; i++) {
+        var currentAttribute = matches[i].split('=');
+        oReturn[currentAttribute[0]] = currentAttribute[1].replace(/["]/g, '');
```

Comment:
these attributes could also be quoted in single quotes `'` according to the [specs](https://www.w3.org/TR/2004/REC-xml-20040204/#sec-prolog-dtd)